pkg/gitreceive/build_test.go

```diff
@@ -0,0 +1,87 @@
+package gitreceive
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"gopkg.in/yaml.v2"
+
+	"github.com/arschles/assert"
+	"github.com/deis/builder/pkg"
+	"github.com/deis/builder/pkg/gitreceive/storage"
+)
+
+const (
+	bucketName = "mybucket"
+	objKey     = "myobj"
+)
+
+func TestGetProcFileFromRepoSuccess(t *testing.T) {
+	tmpDir := os.TempDir()
```

Comment:
I'd suggest [`ioutil.TempDir`](https://godoc.org/io/ioutil#TempDir) instead of `os.TempDir` here. `os.TempDir` returns the standard temp directory on the system, which is always the same, while `ioutil.TempDir` returns a new random directory under that standard dir